server/mergin/commands.py

```diff
@@ -117,6 +127,12 @@ def _check_server():  # pylint: disable=W0612
         else:
             click.secho("Database initialized properly", fg="green")
 
+        if edition == "Community Edition":
+            _check_permissions('/data')
+        else:
+            _check_permissions('/data')
+            _check_permissions('/overviews')
```

Comment:
I would not do this here and move it to maps (maybe as a part of more generic check?), moreover we should not forget that this check would need to be done from celery container not the server one